### PR TITLE
fix(status): allow an empty string of publichPath

### PR DESCRIPTION
**Pull request by hiroppy**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### CODE REVIEW

File: lib/utils/status.js [hunk 1]
```diff
@@ -6,16 +6,16 @@ const runOpen = require('./runOpen');
...
-
   if (options.static && options.static.length > 0) {
+    if (typeof options.static[0].publicPath !== 'undefined') {
+      logger.info(
+        `webpack output is served from ${colors.info(
+          useColor,
+          options.static[0].publicPath
+        )}`
+      );
+    }
```
Comment (alexander-akait) [inline lib/utils/status.js:17]: It is different public paths :smile: I think we can output both, need very helpful message

Comment (hiroppy) [reply lib/utils/status.js:17]: I see, I'll check the `output.publicPath` together.

Comment (alexander-akait) [reply lib/utils/status.js:17]: I think we should keep `options.dev && options.dev.publicPath` and improve static output like:
```
Content not from webpack is served from 'directory' by `publicPath` URL
```

**Comment by alexander-akait** [comment]
Fixed here https://github.com/webpack/webpack-dev-middleware/blob/master/src/options.json

**Pull request closed without merging**